router/config.go

```diff
@@ -764,6 +769,16 @@ func LoadConfig(path string) (*Config, error) {
 		}
 	}
 
```

Comment:
Can we figure this out from the controller instead of needing to set it manually?

Reply:
Or is this one of things where the controller needs to know what the router can do, so the router can't ask the controller? Or more of a temporary thing, where in the long run everything will assume the router is running with the data model?

Reply:
This is a temporary feature flag that is meant to be removed once the HA work is generally available. Without this flag, the routers report to controllers whether they support the new router data model, and the controller sends the appropriate messages.

With this flag, when `false`, the router will never signal to the controller that it supports the new router data model.

This was asked for in order to "limit the blast radius" of bugs that HA changes bring to existing networks. Without it, if people upgrade after this PR is merged, the new router data model code starts to run even if not being used. The end goal is that every network uses routers using the new router data model and legacy ApiSessions/Sessions and messaging on the ctrl channel is removed.